pkg/porter/parameters.go

```diff
@@ -665,7 +663,7 @@ func (p *Porter) resolveParameterSources(ctx context.Context, bun cnab.ExtendedB
 				outputName = source.OutputName
 			case cnab.DependencyOutputParameterSource:
 				// TODO(carolynvs): does this need to take namespace into account
```

Comment:
Yes, but not namespace prob group name?

Reply:
Yes, but I don't know what to do with it yet. If we name every related bundle with a prefix of the group name, this makes it easier to filter on. But we also have to deal with on deletes how do we handle child/parent relationship, esp. if the child has multiple parents?